src/functions-templates/typescript/log/{{name}}.ts

```diff
@@ -1,9 +1,9 @@
 import type { Context } from "https://edge.netlify.com";
```

Comment:
I think we can leave the type import here. `context` is unused in the template itself, but it's good to have around - this is a template, and devs will either use `context` (and be happy to have the types!) or remove it themselves :)

Reply:
Agreed. Another option would be to change the response to include something from `context`, so that the variable is actually used. Something like `context.ip` or `context.geo.city`?

Reply:
I like that idea, a good way for product discovery! Implemented in https://github.com/netlify/cli/pull/5340/commits/30d8ffc0a76bcd7eed52af5201bea221bf7403c1. I chose `context.geo.city`, geolocation is probably more of a mind-blowing feature than the IP address :D